### ISSUE TITLE
[Bug]: Unable to connect to AWS MemoryDB cluster through ssh tunnel

### ISSUE THREAD

**Issue by esigma5** [RedisInsight/RedisInsight#1547]
**Preconditions** (Any important steps we need to know)
RedisInsightV2 installed on MacOS desktop. MemoryDB cluster recenly created. Start an ssh tunnel to an instance on the same vpc with conectivity to the cluster.

**Steps to reproduce** (How to reproduce what you found step by step)

1. Create a memory DB cluster
2. Start an ssh tunnel targetting the cluster's endpoint/port
3. Attempt to create a connection using RedisInsight

**Actual behavior** (A short description of what you found)
- Connection fails with a timeout error

**Expected behavior** (A short description of what you expected to find)
- Should connect

**Screenshots** (Paste or drag-and-drop a screenshot or a link to a recording)

Standard logs
```
12/29/2022, 5:15:15 PM | INFO | ServerService | Getting server info. | {"stack":[]}
12/29/2022, 5:15:15 PM | INFO | SettingsService | Getting application settings. | {"stack":[]}
12/29/2022, 5:15:15 PM | INFO | SettingsService | Succeed to get application settings. | {"stack":[]}
12/29/2022, 5:15:15 PM | INFO | ServerService | Succeed to get server info. | {"stack":[]}
12/29/2022, 5:15:16 PM | INFO | DatabaseService | Getting databases list | {"stack":[]}
12/29/2022, 5:15:16 PM | INFO | SettingsService | Getting application settings. | {"stack":[]}
12/29/2022, 5:15:16 PM | INFO | SettingsService | Succeed to get application settings. | {"stack":[]}
12/29/2022, 5:15:16 PM | INFO | NotificationGateway | Client connected: cvsjEfNnp_DB3BuJAAAD | {"stack":[]}
12/29/2022, 5:15:21 PM | INFO | CaCertificateService | Getting CA certificate list. | {"stack":[]}
12/29/2022, 5:15:21 PM | INFO | ClientCertificateService | Getting client certificates list. | {"stack":[]}
12/29/2022, 5:15:43 PM | INFO | DatabaseService | Creating new database. | {"stack":[]}
12/29/2022, 5:15:43 PM | INFO | RedisService | Successfully connected to the redis database | {"stack":[]}
12/29/2022, 5:16:13 PM | ERROR |  | Request Timeout. POST /api/databases | {"stack":[{}]}
```

Error logs
```
12/29/2022, 5:16:13 PM | ERROR |  | Request Timeout. POST /api/databases | {"stack":[{}]}
```

Desktop screenshot:
![Screenshot 2022-12-29 at 17 25 30](https://user-images.githubusercontent.com/1571636/210007867-ed947c9d-0256-4a02-ad10-abe4048f2013.png)

Using redis-cli client works just fine (so, connectivity from ec2 proxy instance to memorydb is not an issue):
```
redis-cli -c -h localhost  -p 11112 keys \*
(empty array)
``` 

Tried extending the request timeout, but although the timeout was effectively extended, it failed anyway:
```
REQUEST_TIMEOUT=30000 /Applications/RedisInsight-v2.app/Contents/MacOS/RedisInsight-v2
```

**Additional context** (Operating system, version of RedisInsight, Redis database version, Redis module version, database type, connection type, logs, or any other information)

- MacOS Ventura 13.0.1
- RedisInsightV2 Version 2.16.0 (2.16.0.70400)
- SSH tunnel command: ssh  -i ~/.ssh/mykey -L 11112:cluster-dns:6379  ec2-user@ec2-instance-acting-as-jump-server-dns
- No TLS was used
- Tried connecting to a local redis server and it worked fine

**Comment by arthosofteq** [comment]
@esigma5 Could you try on test build from this message? https://github.com/RedisInsight/RedisInsight/issues/1339#issuecomment-1367791335

**Comment by esigma5** [comment]
Hi @arthosofteq , thanks for the response.
I just tried your test build (mac flavor), but sadly the same error arised.

For the record, I tried using the web server version running inside the same VPC where the MemoryDB was located, using [this guide](https://developer.redis.com/explore/redisinsight/autodiscover/) as inspiration, and it worked fine (using the option "Connect to Redis database", because the "Elasticache Database" option did not work).

I will stick with the web version, but of you want me to try any other build, let me know.
